packages/@coorpacademy-player-services/src/types.js

```diff
@@ -64,4 +72,147 @@ type Window = {|
   dataLayer?: Array<DataEvent>
 |};
 
-export type {DataEvent, Resource, ResourceType, Url, Window};
+// type Media = {|
```

Comment:
???

Reply:
not sure yet if this is to be here.
since it's not used so far I keep it commented so we keep the first draft from Faust and see when we oterate on further typing